partnerships/index.md

```diff
@@ -1,11 +1,12 @@
 # Partnerships
 
-Partnerships and sustainability oversees the strategy and systems that 2i2c uses to sustain itself via partnerships with other communities.
-This includes partnerships via paid contracts, as well as via informal and formal collaboration.
+Partnerships oversees the strategy and systems that 2i2c uses to sustain itself via partnerships with other communities.
+This includes partnerships involving paid contracts, as well as via informal and formal collaboration.
 
 ```{toctree}
 overview.md
 structure.md
 workflow.md
-communication.md
+../communication/index
+community_success/freshdesk.md
```

Comment:
Over time we might want to nest more pages under community_success if more content is added there, so I just created a sub-folder for now w/ the freshdesk tips page in it. This is under the assumption that support mechanisms will fall under Partnerships